### PR TITLE
Use full os description in BQ and displays app

### PR DESCRIPTION
**Pull request by tejohnso**
@ahmedalsudani @fjvallarino please review

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ version = requireRoot("version.json"),
 displaySettings;
 
 var externalLogger = require("rise-common-electron").externalLogger
-  (network, platform.getOS(), platform.getArch(), version);
+  (network, platform.getOSDescription(), platform.getArch(), version, platform.getOSDescription());
```
Comment (ahmedalsudani) [inline main.js]: @tejohnso why would we have two parameters if we're going to pass the same argument?

**Comment by tejohnso** [comment]
@ahmedalsudani updated

**Comment by ahmedalsudani** [comment]
:+1:

**Comment by fjvallarino** [comment]
:+1:

**Pull request merged**